CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ Please follow the procedure described in [SECURITY.md](SECURITY.md).
 
 ## CLA
 
-We welcome contributions from everyone but we can only accept them if you sign
+We welcome contributions from everyone but, we can only accept them if you sign
```

Comment:
mm what is the right place for the comma here

```suggestion
We welcome contributions from everyone, but we can only accept them if you sign
```

Reply:
Haha oops. That's what I meant but I mistyped it when I was manually copying the change out of a git stash from a previous week. Thanks.